Tasks/AzureTestPlanV0/Invokers/jestinvoker.ts

```diff
@@ -1,14 +1,39 @@
 import tl = require('azure-pipelines-task-lib/task');
 import utils = require('../utils');
 import constants = require('../constants');
+import tr = require("azure-pipelines-task-lib/toolrunner");
+import { executeJestCommand } from '../testLibExecutor';
 
+//Jest command like: >set JEST_JUNIT_OUTPUT_NAME=TEST-Jest0-junit.xml
+//>npx jest --ci --reporters=default --reporters=jest-junit -t "JestTestName"
 export async function executeJestTests(testsToBeExecuted: string[]): Promise<number> {
 
-    // jest execution will be added
-    /*executable = npm
-    args = test
-    */
-
-    console.log("jest changes1");
-    return 1;
+    //Jest-Junit Link:https://github.com/jest-community/jest-junit
+    let finalStatus = 0; 
+    let npmPath = tl.which("npm", true);
+    await executeJestCommand(npmPath, constants.INSTALL_JESTJUNIT);
```

Comment:
Please handle the installation failure using try catch block and bail out if faced any?

Also, please add debug logs to enable faster investigation as well

Reply:
Added error handling for installation and debug logs